src/test/groovy/com/google/gradle/osdetector/OsDetectorPluginTest.groovy

```diff
@@ -35,7 +35,9 @@ class OsDetectorPluginTest {
     System.err.println('classifier=' + project.osdetector.classifier)
     if (project.osdetector.os == 'linux') {
       assertNotNull(project.osdetector.release.id)
-      assertNotNull(project.osdetector.release.version)
+      if (project.osdetector.release.id != 'arch') {
+        assertNotNull(project.osdetector.release.version)
```

Comment:
Release version is also missing on my debian sid-based system, but meh, it won't be a problem if we set up CI that runs a stable release of linux.